istioctl/pkg/install/pre-check.go

```diff
@@ -149,20 +161,24 @@ func installPreCheck(istioNamespaceFlag string, restClientGetter resource.RESTCl
 			name:      "ConfigMap",
 		},
 	}
-	var errs error
+	var createErrors error
 	resourceNames := make([]string, 0)
+	errResourceNames := make([]string, 0)
 	for _, r := range Resources {
 		err = checkCanCreateResources(c, r.namespace, r.group, r.version, r.name)
 		if err != nil {
+			createErrors = multierror.Append(createErrors, err)
 			errs = multierror.Append(errs, err)
+			errResourceNames = append(errResourceNames, r.name)
 		}
 		resourceNames = append(resourceNames, r.name)
 	}
-	if errs != nil {
-		return errs
+	if createErrors == nil {
+		fmt.Fprintf(writer, "Can create necessary Kubernetes configurations: "+"%v. \n", strings.Join(resourceNames, ","))
```

Comment:
Try not to use concatenation in Printf() and Fprintf().

Reply:
done